cypress.config.mjs

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:8888',
+    // Has to be wide enough for some responsive elements which only render on desktop screens
+    // to render.
+    viewportWidth: 1200,
```

Comment:
I had to make the viewport wide in the cypress environment so that the responsive-ly designed sidebar would appear. It's hidden at smaller viewports.